lib/wovnrb/store.rb

```diff
@@ -200,6 +204,10 @@ def url_pattern
       @settings['url_pattern']
     end
 
+    def custom_domain_langs
+      CustomDomainLangs.new(@settings['custom_domain_langs'])
```

Comment:
should do this once and cache it
something like `@custom_domain_langs ||= CustomDomainLangs.new(@settings['custom_domain_langs'])`

Reply:
Unfortunately this breaks the tests. I guess it's because of `Singleton` which doesn't reset `@custom_domain_langs` and shares it across test cases?

Reply:
It's a hack but our test setup does `Wovnrb::Store.instance.reset`. So you could nil out `@custom_domain_langs` there

Reply:
Fixed in 3d6c13561c876dfc2281ec17b404c1b851bbc635.